src/views/CrontabForm/CrontabForm.tsx

```diff
@@ -0,0 +1,147 @@
+
+export const CronTabForm: React.FC = () => {
+  const [model] = useK8sModel({
+    group: CRONTAB_APIGROUP,
+    version: CRONTAB_APIVERSION,
+    kind: CRONTAB_KIND,
+  });
+  const [name, setName] = useState("");
+  const [cronSpec, setCronSpec] = useState("");
+  const [image, setImage] = useState("");
+  const [replicas, setReplicas] = useState<number | string>(1); // Initialize as 1 or an empty string
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const history = useHistory();
+  const navigate = useNavigate();
+  const { t } = useCronTabTranslation();
+  const [activeNamespace] = useActiveNamespace();
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    setLoading(true);
+    setError("");
+
+    const data = {
+      apiVersion: "stable.example.com/v1",
+      kind: CRONTAB_KIND,
+      metadata: {
+        name,
+        namespace: activeNamespace,
+      },
+      spec: {
+        cronSpec,
+        image,
+        replicas: parseInt(replicas as string, 10),
+      },
+    };
+
+    k8sCreate({ model, data })
+      .then(() => {
+        setLoading(false);
+        history.push(`/k8s/all-namespaces/stable.example.com~v1~CronTab`);
+      })
+      .catch((err) => {
+        setLoading(false);
+        setError(`Error creating CronTab: ${err.message}`);
+      });
+  };
+  return (
+    <>
+      <PageSection>
+        <Title headingLevel="h1">Create Form</Title>
+        <Form onSubmit={handleSubmit}>
+          <FormGroup label="Name" fieldId="crontab-name" isRequired>
+            <TextInput
+              id="crontab-name"
+              name="name"
+              onChange={(e) => setName((e.target as HTMLInputElement).value)}
+              value={name || ""}
+              placeholder="Enter CronTab name"
+              isRequired
+            />
+          </FormGroup>
+
+          <FormGroup label="CronSpec" fieldId="crontab-cronSpec" isRequired>
+            <TextInput
+              id="crontab-cronSpec"
+              value={cronSpec || ""}
+              onChange={(e) =>
+                setCronSpec((e.target as HTMLInputElement).value)
+              }
+              placeholder="e.g., */5 * * * *"
+              isRequired
+            />
+          </FormGroup>
+
+          <FormGroup label="Image" fieldId="crontab-image" isRequired>
+            <TextInput
+              id="crontab-image"
+              value={image || ""}
+              onChange={(e) => setImage((e.target as HTMLInputElement).value)}
+              placeholder="Enter image name"
+              isRequired
+            />
+          </FormGroup>
+
+          <FormGroup label="Replicas" fieldId="crontab-replicas" isRequired>
+            <TextInput
+              type="number"
+              id="crontab-replicas"
+              value={replicas}
+              onChange={(e) =>
+                setReplicas((e.target as HTMLInputElement).value)
+              }
```

Comment:
parseInt here (no need to pass radix, it defaults to 10):
```suggestion
              onChange={(_e, value) =>
                setReplicas(parseInt(value))
              }
```